src/solver/lu_decomposition.cu

```diff
@@ -0,0 +1,177 @@
+#include <iostream> 
+#include <vector>
+#include <micm/util/cuda_param.hpp> 
+const size_t BLOCK_SIZE = 320; 
+struct decomposeDevice{
+    double* A; 
+    double* L; 
+    double* U; 
+    char* do_aik; 
+    size_t* aik; 
+    char* do_aki;
+    size_t* aki;  
+    size_t* uii; 
+    std::pair<size_t,size_t>* niLU;
+    std::pair<size_t, size_t>* uik_nkj; 
+    std::pair<size_t, size_t>* lij_ujk;
+    std::pair<size_t, size_t>* lki_nkj; 
+    std::pair<size_t, size_t>* lkj_uji;
```

Comment:
please update to use a trailing underscore for class/struct data members

Reply:
added trailing lines to all struct members include AddForcing and AddJacobians.